README.md

```diff
@@ -165,6 +165,14 @@ If you would like to support push notification based routing while your app alre
        intent.putExtra("branch_force_new_session",true);
 ```
 
+### Guaranteed Matching
+Branch support hundred percent guaranteed matching with cookie based matching using Custom Chrome Tabs. This is highly recommended if you like to do user authentication through deep link metadata.
+Just add the following to your build.gradle file to enable guaranteed matching
```

Comment:
Let's add a note about Multidex and how to survive it. At least, let's link to the troubleshooting guide where we have that listed.